build-ios-no-tts.sh

```
@@ -163,12 +160,56 @@ libtool -static -o build/os64/sherpa-onnx.a \
+
+  cp $lib_path $fw_dir/SherpaOnnxC
+  cp install/include/sherpa-onnx/c-api/c-api.h $fw_dir/Headers/sherpa-onnx/c-api/
+
+  cat > $fw_dir/Modules/module.modulemap << 'MEOF'
+framework module SherpaOnnxC {
+  header "sherpa-onnx/c-api/c-api.h"
+  export *
+}
+MEOF
+
+  cat > $fw_dir/Info.plist << 'PEOF'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)